docs/Developers/Writing_Native_Bears.rst

```diff
@@ -110,8 +111,8 @@ You should now see an output like this on your command line:
     tutorial, else you will see a bunch of other outputs from other bears
     as well.
 
-The Bear class also supports ``warn`` and ``err`` to create ``WARNING`` and
-``ERROR`` messages respectively.
+For more detail about Python's built-in logging facility,
```

Comment:
Maybe it's better to put logging in double backticks here --> ``logging``. Else LGTM.

Reply:
You are right!